apache-rat-tasks/pom.xml

```diff
@@ -179,25 +185,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
```

Comment:
forced push removed the changes from part1 - can you try merging in master to see if the changes come in?!